floater/utils.py

```diff
@@ -253,12 +253,9 @@ def floats_to_netcdf(input_dir, output_fname,
     float_files = glob(os.path.join(input_dir, match_pattern))
     float_timesteps = sorted(list({int(float_file[-22:-12]) for float_file in float_files}))
 
-    float_columns = ['npart', 'time', 'x', 'y', 'z', 'u', 'v', 'vort']
-    var_names = float_columns[2:]
```

Comment:
Would it be hard to leave this as a fallback if the csv files don't have a header? That would enable it to be backwards compatible with the older output.